packages/copyable/src/Copyable.tsx

```diff
@@ -0,0 +1,283 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import ClipboardJS from 'clipboard';
+import { transparentize } from 'polished';
+
+import Button, { Variant as ButtonVariant } from '@leafygreen-ui/button';
```

Comment:
Ah! I wonder if we should be more proactive about scoping things such as variant/size local to the component, such that we can be more flexible with using these types across components. Probably nothing actionable here in this PR, just thinking